client/app/scripts/components/topology-timestamp-button.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import moment from 'moment';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+
+import { isPausedSelector } from '../selectors/timeline';
+import { TIMELINE_TICK_INTERVAL } from '../constants/timer';
+
+
+class TopologyTimestampButton extends React.Component {
+  componentDidMount() {
+    this.timer = setInterval(() => {
+      if (!this.props.isPaused) {
+        this.forceUpdate();
+      }
+    }, TIMELINE_TICK_INTERVAL);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
+  }
+
+  renderTimestamp() {
+    const { isPaused, updatePausedAt, millisecondsInPast } = this.props;
+    const timestamp = isPaused ? updatePausedAt : moment().utc().subtract(millisecondsInPast);
+
+    return (
+      <time>{timestamp.format('MMMM Do YYYY, h:mm:ss a')} UTC</time>
```

Comment:
Does that mean ISO8601? :D I find it not very readable from a glance, but UI consistency is a valid point. Besides, if we want to make that field editable at some point, it's better to keep it ISO, so I'll change it.